src/main/kotlin/de/cyface/collector/storage/cloud/GoogleCloudStorageService.kt

```diff
@@ -67,7 +67,8 @@ class GoogleCloudStorageService(
         val subscriber = CloudStorageSubscriber<Buffer>(cloud)
 
         targetStream.subscribe(subscriber)
-        pipe.to(targetStream).onSuccess {
+        val pipeToProcess = pipe.to(targetStream)
+        pipeToProcess.onSuccess {
```

Comment:
just out of interest - why is the later better?

Reply:
Makes mocking easier, since I do not have to mock the return value of `onSuccess` and allows to debug, whether pipe.to was mocked correctly.